### PR TITLE
Add Dockerfile that generates pdfs and sets up a nginx server to host the pdfs

### PR DESCRIPTION
**Pull request by Portals**
To easily test it locally, run `docker-compose up --build`, and then navigate to `localhost:8080/reglemente.pdf` for example. The `texfile` image which is used to generate the pdfs is almost 4GB large, so a word of warning there is needed. 

I also fixed some smaller issues which hindered me from running `make all`. 

* Generated pdfs from eps files
* Removed a random newline that broke ekonomisk policy
* Removed styrit14 verksamhetsplan since there were issues there as well. But I don't see any real reason for keeping it. 

Maybe @cthit/digit could set up hub.docker.com and run it in production before merging. 

I tried to get autoindex to work, but I gave up.

### CODE REVIEW

**Comment by ericlp** [comment]
This made autoindex work for me. It feels a bit wrong but it works :shrug: 

```nginx
user  nginx;
worker_processes  1;

events {
}

http {

  autoindex on;
    server {
      listen 8080;
      server_name localhost;
    }

  include /etc/nginx/conf.d/*.conf;
}
```

**Comment by ericlp** [comment]
Serving the files this way will break all links to all dokuments so it would be nice if it is as close to the previous link as possible.
The current path to for example "Reglementet" is https://styrit.chalmers.it/docs/reglemente/reglemente.pdf. 
Would be neat if the nginx server could be hosted on "https://styrit.chalmers.it/docs/" so that the change in links is minimal.

**Comment by Portals** [comment]
I think the issue with it not being at the same URL is a problem for power users. And changing the URLs would not take long.

I envision having all documents under docs.chalmers.it, so if you need reglemente, then you would simply go to `https://docs.chalmers.it/reglemente.pdf`.

**Comment by ericlp** [comment]
Heck, that's kind of reasonable! :+1:

**Comment by ericlp** [comment]
There exists a nginx-module called fancyindex which could be neat but should probably be considered a future enhancement. It allows us to apply css to the index-page and make it less of a sore sight to the eyes.

**Comment by ericlp** [comment]
@cthit/digit Would you be so kind as to look at this and maybe put it up on DockerHub and Rancher?

**Comment by ericlp** [comment]
`README.md` should be updated, for example, it currently states that the documents are hosted at `https://styrit.chalmers.it/docs/`. There should also be directions for how to build the Dockerfile locally.

File: .gitignore [hunk 1]
```diff
@@ -15,7 +15,7 @@ build
 # these rules might exclude image files for figures etc.
 # *.ps
 # *.eps
-*.pdf
+# *.pdf
```
Comment (zimbosaurus) [inline .gitignore:18]: I really like this change!

**Review by zimbosaurus — APPROVED** [review]
I am in favor of this really cool pull request!

**Review by ericlp — APPROVED** [review]

File: classes/cthit_en.cls [hunk 2]
```diff
@@ -179,7 +179,7 @@
         \textbf{Chalmers University of Technology}
 
         \vspace{2cm}
-        \centerline{ \includegraphics[width=0.5\textwidth]{itlogo.eps}}
+        \centerline{ \includegraphics[width=0.5\textwidth]{itlogo.pdf}}
```
Comment (zimbosaurus) [inline classes/cthit_en.cls:182]: This is really cool

**Review by zimbosaurus — APPROVED** [review]

File: classes/cthit_en.cls (hunk 2 above)
Comment (Portals) [reply classes/cthit_en.cls:182]: What are you trying to say?

**Pull request merged**